test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -33,329 +32,42 @@ public void TagName_CanSetToNull()
         }
 
         [Fact]
-        public void GenerateStartTag_ReturnsFullStartTag()
-        {
-            // Arrange
-            var tagHelperOutput = new TagHelperOutput("p", attributes:
-                new Dictionary<string, string>
-                {
-                    { "class", "btn" },
-                    { "something", "   spaced    " }
-                },
-                htmlEncoder: new NullHtmlEncoder());
-
-            // Act
-            var output = tagHelperOutput.GenerateStartTag();
-
-            // Assert
-            Assert.Equal("<p class=\"btn\" something=\"   spaced    \">", output);
-        }
-
-        [Fact]
-        public void GenerateStartTag_ReturnsNoAttributeStartTag()
+        public void CanSetPreContent()
```

Comment:
- `PreContent_SetContent_ChangesValue` and same for other new tests.  ("can" isn't an expectation and it should not appear in test names.)
- better to confirm all `TagHelperContent` properties are non-`null` in a newly-initialized `TagHelperOutput` instance.  these are currently `TagHelperContent` tests that assume that is true.